src/openforms/contrib/objects_api/setup_configuration/steps.py

```diff
@@ -0,0 +1,58 @@
+from django_setup_configuration.configuration import BaseConfigurationStep
+from zgw_consumers.models import Service
+
+from openforms.contrib.objects_api.models import ObjectsAPIGroupConfig
+
+from .models import ObjectsAPIGroupConfigModel, SingleObjectsAPIGroupConfigModel
+
+
+def get_service(slug: str) -> Service:
+    """
+    Try to find a Service and re-raise DoesNotExist with the identifier to make debugging
+    easier
+    """
+    try:
+        return Service.objects.get(slug=slug)
+    except Service.DoesNotExist as e:
+        raise Service.DoesNotExist(f"{str(e)} (identifier = {slug})")
+
+
+class ObjectsAPIConfigurationStep(BaseConfigurationStep[ObjectsAPIGroupConfigModel]):
+    """
+    Configure configuration groups for the Objects API backend
+    """
+
+    verbose_name = "Configuration to set up Objects API registration backend services"
```

Comment:
I think this `verbose_name` is only used in command output (https://github.com/maykinmedia/django-setup-configuration/blob/7cf1e39383b8be9a123f440d9b7e215cada575e2/django_setup_configuration/management/commands/setup_configuration.py#L68), so I don't think so?

I could check with Sidney next week to ask if he thinks this is needed?

Reply:
okay - I'm a little bit concerned because everywhere else in django `verbose_name` is typically marked translatable and people looking at this code can get the wrong impression that they don't have to mark strings as translatable, but that's a task for reviewers to watch out for then :)